.github/workflows/test-python-package.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.6, 3.7, 3.8, 3.9, 3.10]
```

Comment:
I believe `3.10` requires  quotes around it: https://github.com/actions/setup-python/issues/160